modules/file-integrity-understanding-file-integrity-node-statuses-object.adoc

```diff
@@ -24,7 +24,7 @@ worker-fileintegrity-ip-10-0-165-160.ec2.internal   102s
 
 [NOTE]
 ====
-The `FileIntegrityNodeStatus` object might not be created until the second run of the scanner is finished. The period is configurable.
+The `FileIntegrityNodeStatus` object might not be created until the second run of the scanner is finished.
```

Comment:
@rhmdnd I couldn't find anything in the [upstream docs](https://github.com/openshift/file-integrity-operator) about any of this. We could optionally remove this entire note. I find it rather confusing as well.

Reply:
Yeah - perhaps just a disclaimer that the scan results show up some time after, but not immediately?